ingest/Maybe.py

```diff
@@ -0,0 +1,55 @@
+__author__ = 'Stephan'
+
+from itertools import chain
+
+class Maybe:
+    def __init__(self, v=None):
+        self.value = v
+
+    @staticmethod
+    def nothing():
+        return Maybe()
+
+    @staticmethod
+    def of(t):
```

Comment:
Ahh sorry I thought you meant "do we need to have comments OUT any/all of these?" but actually you mean "do we need to have comments on any/all of these?".. Sorry I misread that in a glimpse!

Yes we should have a bit more comment in the Maybe class! I have an exam this afternoon so might not be able to complete this within today though.